pkg/commands/drift/command.go

```diff
@@ -122,20 +166,15 @@ func (c *DetectIamDriftCommand) Run(ctx context.Context, args []string) error {
 		return fmt.Errorf("failed to detect drift %w", err)
 	}
 
-	// Output to stdout to mimic bash script for now.
-	// TODO(dcreey): Determine cleaner API that aligns with using the cli tool.
-	if len(iamDiff.ClickOpsChanges) > 0 {
-		uris := keys(iamDiff.ClickOpsChanges)
-		sort.Strings(uris)
-		c.Outf("Found Click Ops Changes \n> %s", strings.Join(uris, "\n> "))
-		if len(iamDiff.MissingTerraformChanges) > 0 {
-			c.Outf("\n\n")
-		}
+	if len(iamDiff.ClickOpsChanges) > 0 || len(iamDiff.MissingTerraformChanges) > 0 {
+		c.Outf(driftMessage(iamDiff))
 	}
-	if len(iamDiff.MissingTerraformChanges) > 0 {
-		uris := keys(iamDiff.MissingTerraformChanges)
-		sort.Strings(uris)
-		c.Outf("Found Missing Terraform Changes \n> %s", strings.Join(uris, "\n> "))
+
+	if !c.flagSkipGitHubIssue {
+		err = createUpdateOrCloseIssues(ctx, c.flagGitHubToken, c.flagGitHubOwner, c.flagGitHubRepo, c.flagGitHubIssueAssignees, c.flagGitHubIssueLabels, iamDiff)
```

Comment:
This seems to be a "junk drawer" function as evidenced by both it's name and the number of parameters. Does it make sense for this to be it's own struct with its own logic/encapuslation?

Reply:
Refactored some.